modules/sievefilters/modules.php

```diff
@@ -1179,9 +1216,6 @@ protected function output() {
         $res .= get_script_modal_content();
         $res .= '<div class="p-3">';
         foreach($mailboxes as $idx => $mailbox) {
-            if (empty($mailbox['sieve_config_host'])) {
```

Comment:
This condition remains important, isn't it?

Reply:
Yes